histocc/formatter/hisco.py

```diff
@@ -388,3 +391,9 @@ def blocky5() -> BlockyHISCOFormatter: # pylint: disable=C0116
     )
 
     return formatter
+
+
+def blocky5() -> BlockyHISCOFormatter: # pylint: disable=C0116
+    warnings.warn(DeprecationWarning('`blocky5` is being deprecated in favor of `hisco_blocky5`'))
+
```

Comment:
Nice detail!

Reply:
Looks good. We should still consider ways of making the formatter construction user friendly. But this seems like one important step in this direction!